bootstrap/bungeecord/src/main/java/org/geysermc/platform/bungeecord/GeyserBungeePlugin.java

```diff
@@ -171,6 +172,16 @@ public BootstrapDumpInfo getDumpInfo() {
         return new GeyserBungeeDumpInfo(getProxy());
     }
 
+    @Override
+    public Path getLogsPath() {
+        Path getFork = Paths.get("proxy.log.0");
```

Comment:
Check for BungeeCord fork status first, then get this path. I.E. `if (isBungeeCord) Paths.get(proxylog) else Paths.get(latest.log)`.